### PR TITLE
new todo-frontend

### PR DESCRIPTION
**Pull request by gjbianco**

### CODE REVIEW

File: todo-frontend/Dockerfile [hunk 1]
```diff
@@ -1,27 +0,0 @@
-FROM registry.access.redhat.com/ubi8:8.0
```
Comment (rsriniva) [inline todo-frontend/Dockerfile:1]: Think we need a Dockerfile/Containerfile for the FE container

**Review by rsriniva — CHANGES_REQUESTED** [review]
All good. app should be tested on Node.js version on classroom workstation VM. Check with richard to see what rpm version in our repos is- I suspect 10.x. All node.js labs should consistently use the same node/npm version - preferably latest LTS v14?

**Review by rsriniva — APPROVED** [review]
Approved. Merge. I will QA when complete lab is ready

**Comment by richardallred** [comment]
should this have gone into `main`?

**Pull request merged**